src/datadoc/frontend/components/variables_tab.py

```diff
@@ -36,8 +36,8 @@ def build_variables_tab() -> dbc.Tab:
                         ssb.Input(
                             label="Søk i variabler",
                             searchField=True,
-                            disabled=True,
-                            placeholder="Kommer...",
+                            disabled=False,
+                            placeholder="Variabel kortnavn",
```

Comment:
It is common with ... after text to emphasize the text is only placeholder and will be removed